helpers/skills/vllm-compare-reqs/SKILL.md

```
@@ -44,47 +44,50 @@ Use this skill when you need to:
+./scripts/compare_reqs.py v0.19.0 v0.20.2 rocm   # common.txt + rocm.txt + build/rocm.txt + Dockerfiles
+./scripts/compare_reqs.py v0.19.0 v0.20.2 cuda   # common.txt + cuda.txt + build/cuda.txt + Dockerfile
+./scripts/compare_reqs.py v0.19.0 v0.20.2 cpu    # common.txt + cpu.txt + build/cpu.txt + Dockerfile.cpu
+./scripts/compare_reqs.py v0.19.0 v0.20.2 tpu    # common.txt + tpu.txt + Dockerfile.tpu
+./scripts/compare_reqs.py v0.19.0 v0.20.2 xpu    # common.txt + xpu.txt + Dockerfile.xpu (Intel GPU)
 ```
 
 ## Output Format
 
 The script provides clean, categorized output with a **summary table** followed by detailed changes:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)